### DISCUSSION TITLE
push directive notwork livewire3

### DISCUSSION THREAD

**Discussion by zsnakeee** [livewire/livewire#6248] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Additional information

I am encountering an issue with the Livewire full-page component, where the push directive is not functioning as expected. Specifically, the push directive is not pushing to the stack of the blade template. It appears to be working correctly in Livewire v2.

Component Code
`return view('livewire.home')->layout('layouts.app');`

Blade Code
`
@push('js')
    <script>
        console.log('home.blade.php')
    </script>
@endpush
`

Layout Code
`@stack('js')`

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment]
Some PR's for this were already created.
Failing test: https://github.com/livewire/livewire/pull/6135
Passing test: https://github.com/livewire/livewire/pull/6201
